components/financial_assistance/app/domain/financial_assistance/operations/applications/medicaid_gateway/add_mec_check.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+require 'aca_entities/magi_medicaid/libraries/iap_library'
+
+module FinancialAssistance
+  module Operations
+    module Applications
+      module MedicaidGateway
+        # This Operation adds the MEC Check to the Application(persistence object) and Applicants
```

Comment:
comments in this file need to be fixed. take out references to persistence object and applicants